src/components/member-page/donation/index.js

```diff
@@ -87,6 +105,12 @@ const MemberDonationPage = () => {
   const [records, setRecords] = useState([])
   const [showEmptyState, setShowEmptyState] = useState(false)
   const [isLoading, setIsLoading] = useState(true)
+  const [isYearlyReceiptDownloading, setIsYearlyReceiptDownloading] = useState(
+    false
+  )
+  const [yearlyDownloadTextYear, setYearlyDownloadTextYear] = useState(2024)
```

Comment:
it seems would would always show `2024年度收據` before getting current year for the download button. this may become a bit confusing if the year is not 2024.
How about hidding the download button until the correct year data retrieved ?

Reply:
現在是為了 Demo 才這麼做的，之後會等 api 回傳是否有年度收據，再決定要不要顯示按鈕，到時候在一起顯示年份和按鈕。

Reply:
okay